packages/api/app/Middleware/Role.js

```diff
@@ -0,0 +1,15 @@
+const InvalidAccessException = use('App/Exceptions/InvalidAccessException');
+const RoleModel = use('App/Models/Role');
+class Role {
+	async handle({ auth }, next, properties) {
+		const user = await auth.getUser();
+		await user.load('role');
+		const userRole = user.toJSON().role.role;
```

Comment:
I think something like that it's possible:

```js
const userRole = await auth.user.role().role;
```

Reply:
Não funciona...